src/app/models/user.ts

```diff
@@ -0,0 +1,8 @@
+import { Hero } from './hero';
+
+export class User {
```

Comment:
why do you need this for the navbar?

Reply:
In early stage I wanted to populate heroes hall with dummy data from user, because this holds the hero array but later on never used. Still kept with all the other models.